torch/csrc/cuda/Event.cpp

```diff
@@ -89,8 +89,7 @@ static PyObject* THCPEvent_from_ipc_handle(
   }
   THCPEvent* self = (THCPEvent*)ptr.get();
 
-  // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
-  cudaIpcEventHandle_t handle;
+  cudaIpcEventHandle_t handle{};
```

Comment:
I am not sure these should be changed since I have no idea if the ctor is trivial to initialize or not (or what will happen when it's memcpyied over it

Reply:
cudaIpcEventHandle_t is a pointer, the change sets it to nullptr.